src/styles/main.scss

```diff
@@ -1,7 +1,13 @@
-@import "utils";
-@import "fonts";
-@import "typography";
-
-body {
```

Comment:
The 'body' styles are missing from your main.scss file. If these styles are necessary for your layout, you should include them. Consider adding 'body { background: $c-gray; }' back to your code.